.ci/windows-steps.yaml

```diff
@@ -78,8 +76,7 @@ steps:
     msbuildVersion: $(MSBuildVersion)
     configuration: 'Release'
     msbuildArchitecture: 'x64'
-    msbuildArguments: /m /p:BuildInParallel=true
-    maximumCpuCount: false
+    maximumCpuCount: 1
```

Comment:
Maybe adding `msbuildArguments: /Zm10` solves the issue?

Reply:
See https://docs.microsoft.com/en-us/cpp/build/reference/zm-specify-precompiled-header-memory-allocation-limit?view=vs-2019 for more information, I guess you can also directly go for `/Zm200`.

Reply:
I will try this one too.

Reply:
@shrit want to try the `/Zm10` or `/Zm200` options and see if they fix the remaining C1060 errors?  We are getting close---only `feedforward_network_test.cpp` and `recurrent_network_test.cpp` have problems now, and I know those particular files have very large RAM usage on Linux too.

Reply:
In the worst case, we could perhaps split those two files into multiple files, in the hopes that smaller files would have smaller RAM usage.